### PR TITLE
Creating a data model

### PR DESCRIPTION
**Pull request by Feliverse**
### Project requirements
- Create a new app. Give it a meaningful name.
- Initialize your project with Git.
- Make sure that your project has Postgres database set up.
- Build your project schema.
    - Create and run the necessary migration files.
    - Table and column names should match the ERD diagram.
        - note: photo for users table should be a link to an image
    - Foreign keys should be included.
        - Pay attention that in `posts` table the column holding the foreign key to the `users` table should be named `author_id`
    - All columns that are foreign keys should have a corresponding index.

### CODE REVIEW

**Review by TadesseDev — CHANGES_REQUESTED** [review]
### Hi @Feliverse 

### I'm Tadesse Alemayehu your reviewer for this PR. :hand: 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request 🛑🛑
yPkfDFf

 - [x] I see you have an empty readme file, This will affect your professionality as a developer. We need to use a professional readme file in all our projects and for that [Microverse provides a Readme template](https://github.com/microverseinc/readme-template) we can customize and use it. I suggest you do the same before the next review request.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: db/schema.rb [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+  create_table "users", force: :cascade do |t|
+    t.string "Name"
+    t.binary "Photo"
+    t.text "Bio"
+    t.integer "PostsCounter"
+    t.datetime "created_at", null: false
+    t.datetime "updated_at", null: false
+  end
+
+end
```
Comment (DammyShittu) [inline db/schema.rb]: - [x] Kindly stick to the naming conventions for your table which is lowercase and underscore for separating names. Kindly rename these columns to `name`, `photo`, `bio` and `posts_counter`.

- [x] Your database is missing the other defined tables in the diagram. Kindly create them

  https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/images/blog_app_erd_v1_1.png

- [x] **[OPTIONAL]** I noticed you generated your views and so on with scaffold. I strongly recommend you remove any files generated by scaffold as that would mean creating a lot of unnecessary files that would not be used.

**Review by DammyShittu — CHANGES_REQUESTED** [review]
Hi @Feliverse,

## Highlights
✔️ Postgres for db

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: db/schema.rb (hunk 1 above)
Comment (Feliverse) [reply db/schema.rb]: Hi @DammyShittu ,  I notice, proceeding changes, see you

**Review by KDlamini — APPROVED** [review]
### Hi @Feliverse 🙋‍♀️,  

Excellent work! 💯 
You have done a great job implementing all the required changes. 🎊 🥳 
Your feature is complete! There is nothing else to say other than... it's time to merge it. :shipit: 
Congratulations!!!

## HIGHLIGHTS 🌟 

✔️ Good PR title and description
✔️ Migrations files for users, posts, comments, and likes are created
✔️ Foreign keys with indexes are implemented where applicable
✔️ Table attributes follow the ERD diagram.

# Status: Approved ✔️ 

Great-job-smiley

Cheers 🥂  and Happy coding!!! 👯 
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me @KDlamini  in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**